specification.rst

```diff
@@ -2663,21 +2689,22 @@ speech
     Use ``container-titlee`` if the presentation is part of a larger session
     (e.g. a paper in a symposium);
     For papers published in conference proceedings, use ``paper-conference``;
-    For artistic performance, use ``performance``
+    For artistic performances, use ``performance``
 
 thesis
-    A thesis written for an educational institution to satisfy the requirements
-    for a degree;
+    A thesis written to satisfy requirements for a degree;
     Use ``genre`` to specify the type of thesis
 
 treaty
-    A treaty agreement between two political authorities
+    A treaty agreement between political authorities
```

Comment:
strike "between" and replace with "among"

Reply:
FYI, you can propose changes like @adam3smith did. That would give me a button to click to just accept the proposal

Reply:
Did what I just do result in that button?

Wasn't super intuitive.

Reply:
You made a direct change to the file (the ability for maintainers to do that is on by default in PRs in this repo). 

What Sebastian did was to "propose a change".